cmd/mock-es/main.go

```diff
@@ -127,7 +127,7 @@ func loggingMiddleware(next http.Handler) http.Handler {
 			return
 		}
 
-		log.Printf("%s %s\n%s", r.Method, r.URL.RequestURI(), body)
+		log.Printf("%s %s\n", r.Method, r.URL.RequestURI())
```

Comment:
I think we can use `http.DetectContentType(body)` here and if the body is text/plain or JSON we log it, otherwise we can print `<compressed>` or something similar.